src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.tsx

```diff
@@ -43,17 +61,48 @@ class ProjectKeyCredentials extends React.Component<Props> {
     return (
       <React.Fragment>
         {showDsnPublic && (
-          <Field label={t('DSN')} inline={false} flexibleControlStateSize>
+          <Field
+            label={t('DSN')}
+            inline={false}
+            flexibleControlStateSize
+            help={tct('The DSN tells the SDK where to send the events to. [link]', {
+              link: showDsn ? (
+                <Link onClick={this.toggleDeprecatedDsn}>
+                  {showDeprecatedDsn
+                    ? t('Hide deprecated DSN')
+                    : t('Show deprecated DSN')}
+                </Link>
+              ) : null,
+            })}
+          >
             <TextCopyInput>
               {getDynamicText({
                 value: data.dsn.public,
                 fixed: '__DSN__',
               })}
             </TextCopyInput>
+            {showDeprecatedDsn && (
+              <StyledField
+                label={null}
+                help={t(
+                  "Deprecated DSN includes the secret which is no longer required by Sentry' newer versions of SDKs. If you are unsure which to use, follow installation instructions for your language."
```

Comment:
```suggestion
                  "Deprecated DSN includes a secret which is no longer required by newer SDK versions. If you are unsure which to use, follow installation instructions for your language."
```

Slightly shorter and I found the Sentry part awkwardly placed in the sentence.

Reply:
Thank you, I've updated it both in here and in the original place.